homeassistant/components/sensor/zwave.py

```diff
@@ -7,15 +7,27 @@
 at https://home-assistant.io/components/zwave.html
 """
 # pylint: disable=import-error
+from homeassistant.helpers.event import track_point_in_time
 from openzwave.network import ZWaveNetwork
 from pydispatch import dispatcher
-
+import datetime
+import homeassistant.util.dt as dt_util
 import homeassistant.components.zwave as zwave
 from homeassistant.helpers.entity import Entity
 from homeassistant.const import (
     ATTR_BATTERY_LEVEL, STATE_ON, STATE_OFF,
     TEMP_CELCIUS, TEMP_FAHRENHEIT, ATTR_LOCATION)
 
+PHILIO = '013c'
+PHILIO_SLIM_SENSOR = '0002'
+PHILIO_SLIM_SENSOR_MOTION = (PHILIO, PHILIO_SLIM_SENSOR, 0)
```

Comment:
We should include the command class in this check, as there are two different command classes handled by this sensor platform.